electrode-ota-server-routes-acquisition/src/routes.js

```diff
@@ -18,7 +18,7 @@ export default (options, route, acquisition, logger, ccm) => {
         },
         {
             method: "GET",
-            path: "/update_check",
+            path: "/v0.1/public/codepush/update_check",
```

Comment:
this is going to break older versions?

Reply:
nope, the older versions are already there. only the new services paths are updated here.

Reply:
so you are saying we remove the "/update_check" as no one will call it.  And only use "/v0.1/public/codepush/update_check".